### DISCUSSION TITLE
Which version of NeuroDesktop should users download by default?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#131] [General]
Right now the installation instructions for Linux https://neurodesk.github.io/docs/neurodesktop/getting-started/linux/ suggest downloading version 20210929 of NeuroDesktop, which is not the latest version.
Do we want to:
1) have the documentation automatically point to the latest version
2) suggest people to download vnmd/neurodesktop:latest
3) flag only some versions as "stable", and suggest the users to download the latest stable version
I guess that in the future we will have a parallel development branch, but I wonder what will be the best solution for now?

**Comment by aswinnarayanan** [comment]
Hi @civier. By default, users should download the version of neurodesktop specified in the docs (currently 20210929) 
This would be latest build that has been tested manually. 

vnmd/neurodesktop:latest would be the latest automated build.

**Reply by aswinnarayanan** [reply]
We could tag the version specified in the docs as stable.
And leave latest for the latest automated build

**Reply by stebo85** [reply]
possible, but I don't see much benefit in this

**Comment by stebo85** [comment]
Dear @civier,

20210929 is currently the latest tested version. As Aswin wrote: The documentation always points to the latest tested version.

1) have the documentation automatically point to the latest version
-> this is currently the case! The version is automatically updated everywhere once it was manually tested 
it can be set here: https://github.com/NeuroDesk/neurodesk.github.io/blob/hugo-docsy/data/neurodesktop.toml
-> The release process is documented here: https://neurodesk.github.io/developers/architecture/release_process/

2) suggest people to download vnmd/neurodesktop:latest
-> NO! This will give people an unreproducible environment as the version they run will depend on the time they run the command! Docker will only update latest on an explicit pull command - not on run!

3) flag only some versions as "stable", and suggest the users to download the latest stable version
I guess that in the future we will have a parallel development branch, but I wonder what will be the best solution for now?
-> We already have a development branch: https://hub.docker.com/repository/docker/vnmd/neurodesktop-dev
-> Stable is what's documented in it can be set here: https://github.com/NeuroDesk/neurodesk.github.io/blob/hugo-docsy/data/neurodesktop.toml
